.github/workflows/gradle.yml

```diff
@@ -41,3 +41,8 @@ jobs:
         run: chmod +x gradlew
       - name: Test all reference implementations with Gradle
         run: ./gradlew test --continue
+      - name: Create test summary
+        uses: test-summary/action@dist
```

Comment:
Could you refer to a specific SHA instead of a branch?

Reply:
Good catch, I copied it from the README without thinking... Fixed 👍